mozilla/fx-private-relay
# Add Dashboard Page [2/3] - Caller and SMS Sender Panel

# New feature description

This adds the Callers and SMS Senders log subpage/view that the user can navigate to within the Phone Dashboard UI.


<img width="930" alt="image" src="https://user-images.githubusercontent.com/13066134/185474535-f771ae79-b6b4-4c7e-addb-21734d6ed42d.png">

# How to test

On the Phone Dashboard page, select 'Caller and SMS senders' on the top right of the dashboard card and a list of senders should show. Check that each sender can be individually blocked by clicking its 'Block' button, messages should not be able to through this number if it's in a blocked state.

# Remaining to-do
- [x] Place strings in `pendingTranslations.ftl`
- [x] Add call/text type UI icon indicator (waiting on https://github.com/mozilla/fx-private-relay/pull/2346)
- [x] Microinteractions: hover/focus states
- [x] Mobile UI
- [ ] Panel view when Caller & SMS Senders is disabled (will open up another PR for this one)

# Checklist
